embassy-nxp/Cargo.toml

```diff
@@ -79,6 +79,6 @@ _rt1xxx = []
 _time_driver = ["dep:embassy-time-driver", "dep:embassy-time-queue-utils"]
 
 #! ### Chip selection features
-lpc55 = ["dep:lpc55-pac"]
+lpc55 = ["nxp-pac/lpc55s69_cm33_core0"]
```

Comment:
Do we want to rename the feature to match the fact that it is core0? (core1 obviously happen for a bit I suppose).